projects/cm_mcu/cm_mcu.c

```diff
@@ -342,9 +344,12 @@ int main(void)
   xEPRMQueue_out = xQueueCreate(5, sizeof(uint64_t));
   configASSERT(xEPRMQueue_out != NULL);
 
-  tempAlarmTask.xAlmQueue = xQueueCreate(10, sizeof(uint32_t)); // ALARM queue
+  tempAlarmTask.xAlmQueue = xQueueCreate(10, sizeof(uint32_t)); // temperature ALARM queue
```

Comment:
We should probably remove all references to temperature in the alarm queue and in the messages sent, as we discussed, since we want to use one queue for both voltage and temperature alarms.